language/move-lang/src/typing/translate.rs

```diff
@@ -1780,6 +1780,17 @@ fn builtin_call(
 ) -> (Type, T::UnannotatedExp_) {
     use N::BuiltinFunction_ as NB;
     use T::BuiltinFunction_ as TB;
+    match nb_ {
+        NB::MoveTo(_) |
+        NB::MoveFrom(_) |
+        NB::BorrowGlobal(_, _) |
+        NB::Exists(_) => {
+            if context.current_module.is_none() {
+                context.error(vec![(bloc, "Global storage operator cannot be used from a 'script' function")]);
+            }
+        }
```

Comment:
I think it might make more sense to put this in globals.rs Since you have to then comment that there is already an error reported there. Could we just move there error over to where that comment is?

Reply:
Great idea! I had originally implemented this in translate.rs and then only discovered the code in globals.rs when I hit a test failure. I've updated to do it that way, and it's definitely simpler that way.